flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/PrometheusPushGatewayReporterOptions.java

```diff
@@ -50,4 +50,11 @@
 		.key("deleteOnShutdown")
 		.defaultValue(true)
 		.withDescription("Specifies whether to delete metrics from the PushGateway on shutdown.");
+
+	public static final ConfigOption<Boolean> FILTER_LABEL_VALUE_CHARACTER = ConfigOptions
+		.key("filterLabelValueCharacters")
+		.defaultValue(true)
+		.withDescription("Specifies whether to filter label value characters."
+			+ " It's default to true, so only [a-zA-Z0-9:_] are valid."
```

Comment:
The default doesn't have to be referenced in the description.

Replace this call with:
```
			.withDescription(Description.builder()
				.text("Specifies whether to filter label value characters." +
					"If enabled, all characters not matching [a-zA-Z0-9:_] will be removed," +
					"otherwise no characters will be removed." +
					"Before disabling this option please ensure that your" +
					"label values meet the %s.", LinkElement.link("https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels", "Prometheus requirements"))
				.build());
```

Reply:
thanks for your advice, update and re-generating document...